plugins/themes/default/DefaultThemePlugin.inc.php

```diff
@@ -169,6 +169,9 @@ public function init() {
 
 		// Load custom JavaScript for this theme
 		$this->addScript('default', 'js/main.js');
+
+		// Adding default navigationMenuAreas for all themes
+		$this->addOption('navigationMenuArea', null, array('primary', 'user', 'footer'));
```

Comment:
Remove `footer` as the default theme doesn't support it.

Reply:
OK